README.md

```diff
@@ -136,6 +136,12 @@ Select the subscription that will be used for the deployment:
 azd env set AZURE_SUBSCRIPTION_ID $AZURE_SUBSCRIPTION_ID
 ```
 
+Set your principal name:
```

Comment:
(Optional) Set your principal name.  

This should an optional task for them.  If not set use a standard name like “Developer”

Reply:
done